app/qml/pages/InfoPage.qml

```diff
@@ -71,7 +71,7 @@ Page {
             horizontalAlignment: Text.AlignHCenter
             wrapMode: Text.WrapAtWordBoundaryOrAnywhere
             //TRANSLATORS: Please make sure the URLs are correct
-            text: i18n.tr("Released under the terms of the <a href=\"https://github.com/AsteroidOS/telescope/blob/master/LICENSE\">GNU GPL v3</a>")
+            text: i18n.tr("Copylefted libre software licensed <a href=\"https://github.com/AsteroidOS/telescope/blob/master/LICENSE\">GPLv3+</a>")
```

Comment:
I like this :+1:

Reply:
I agree with the general concept because it is more descriptive. I would suggest a comma behind "software".  So instead of "`... software licensed GPLv3`" I would write "`... software, licensed GPLv3`".